girder/utility/shell.py

```diff
@@ -42,7 +42,8 @@ def _launchShell(context):
         return code.interact(banner=header, local=context)
 
 
-@click.command(context_settings=dict(help_option_names=['-h', '--help']))
+@click.command('shell', short_help='Run a Girder shell.',
+               context_settings=dict(help_option_names=['-h', '--help']))
```

Comment:
Why is `help_option_names` here? If there's a good reason, can you add a code comment? I realize you didn't add this here, but it's confusing as-is.